c2corg_ui/static/js/feed.js

```diff
@@ -114,16 +163,165 @@ app.FeedController = function(appAuthentication, appApi, appLang, imageUrl, ngeo
+
+  this.hasAnnounce = true;
+  this.announce = {'title':'','message': ''};
+
+};
+
+/**
+ * toggle block above forum topics list
+ @param {number} id
+ * @export
+ */
+app.FeedController.prototype.toggleBlock = function(id) {
+  switch (id) {
+    case 0:
+      this.showAssoBlock = !this.showAssoBlock;
+      window.localStorage.setItem('showAssoBlock', JSON.stringify(this.showAssoBlock));
+      break;
+    case 1:
+      this.showMobileBlock = !this.showMobileBlock;
+      window.localStorage.setItem('showMobileBlock', JSON.stringify(this.showMobileBlock));
+      break;
+    default:
+      break;
+  }
+};
+
+/**
+ * init array for the column Manager
+ * @private
+ */
+app.FeedController.prototype.initDocumentsCol_ = function() {
+  if (this.documentsCol[0] === null || this.documentsCol[0] == undefined) {
+    this.documentsCol[0] = Array();
+  }
+  if (this.documentsCol[1] === null || this.documentsCol[1] == undefined) {
+    this.documentsCol[1] = Array();
+  }
+  if (this.documentsCol[2] === null || this.documentsCol[2] == undefined) {
+    this.documentsCol[2] = Array();
+  }
+};
+
+/**
+ * refresh the feed column according the width
+ * @export
+ */
+app.FeedController.prototype.feedColumnManager = function() {
+
+  $(window).resize(function() {
+
+    if (window.innerWidth < 1400) {
+      if (this.nbCols_ != 1) {
+        this.documentsCol = Array();
+        this.documentsCol[0] = this.documents;
+        this.documentsCol[1] = Array();
+        this.documentsCol[2] = Array();
+
+        this.nbCols_ = 1;
+        this.scope_.$apply();
+      }
+
+    } else if (window.innerWidth >= 1400 && window.innerWidth < 2000) {
+
+
+      if (this.nbCols_ != 2) {
+        this.documentsCol = Array();
+        this.documentsCol[0] = Array();
+        this.documentsCol[1] = Array();
+        this.documentsCol[2] = Array();
+        this.nbCols_ = 2;
+
+        var height1_c2 = 0;
+        var height2_c2 = 0;
+
+        for (var i = 0,n = this.documents.length; i < n; i++) {
+
+          if (height1_c2 <= height2_c2) {
+            this.documentsCol[0].push(this.documents[i]);
+            height1_c2 = height1_c2 + this.sizeEstimator(this.documents[i]);
+          } else if (height2_c2 <= height1_c2) {
+            this.documentsCol[1].push(this.documents[i]);
+            height2_c2 = height2_c2 + this.sizeEstimator(this.documents[i]);
+          } else {
+            this.documentsCol[0].push(this.documents[i]);
+            height1_c2 = height1_c2 + this.sizeEstimator(this.documents[i]);
+          }
+
+
+        }
+        this.scope_.$apply();
+      }
+    } else {
+      if (this.nbCols_ != 3) {
+        this.documentsCol = Array();
+        this.documentsCol[0] = Array();
+        this.documentsCol[1] = Array();
+        this.documentsCol[2] = Array();
+        this.nbCols_ = 3;
+
+        var height1_c3 = 0;
+        var height2_c3 = 0;
+        var height3_c3 = 0;
+
+        for (var j = 0,o = this.documents.length; j < o; j++) {
```

Comment:
You can use the same variable name than in the above for loop ("i" and "n"), since the scope of those variables is the for loop block itself (no conflict with vars of the other for loops).

Reply:
i have a travis error with the same name.